keyboards/mschwingen/modelm/keymaps/default/keymap.c

```diff
@@ -52,27 +52,25 @@ const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
     // Cursor: Media Pref / Next / Volume Up / Volume Down
     // Space:  Media Play / Pause
     // m: enter mouse layer
-    ______,  DYN_MACRO_PLAY1, DYN_MACRO_PLAY2, DYN_REC_START1, DYN_REC_START2, DYN_REC_STOP, ______, ______ , ______ , ______, ______, ______, ______, ______, ______, ______,
-    KC_LOCK, ______, ______, ______, ______, ______ , ______ , ______ , ______ , ______ , ______ , ______ , ______ , ______ , ______ , ______, ______ ,  ______ , ______ , ______ , ______ ,
-    ______,  ______, ______, ______, ______, ______ , ______ , ______ , ______ , ______ , ______ , ______ , ______ , ______ , ______ , ______, ______ ,  ______ , ______ , ______ , ______ ,
-    KC_CAPS, ______, ______, ______, ______, ______ , ______ , ______ , ______ , ______ , ______ , ______ , ______ , ______ ,                            ______ , ______ , ______ ,
-    ______,  ______, ______, ______, ______, ______ , ______ , ______ , TG(_MS), ______ , ______ , ______ ,           ______ ,                    KC_AUDIO_VOL_UP,       ______ , ______ , ______ , ______ ,
-    ______,          ______,                          KC_MEDIA_PLAY_PAUSE ,                        ______ ,          ______ , KC_MEDIA_PREV_TRACK , KC_AUDIO_VOL_DOWN , KC_MEDIA_NEXT_TRACK , ______ ,          ______
+    _______, DM_PLY1, DM_PLY2, DM_REC1, DM_REC2, DM_RSTP, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
```

Comment:
Some white-space changes to make the keymap more clear here...

```suggestion
    _______,          DM_PLY1, DM_PLY2, DM_REC1, DM_REC2, DM_RSTP, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
```